specs/order_spec.rb

```diff
@@ -76,35 +76,91 @@
       result.must_equal true
     end
   end
+
+  describe "remove_product" do
+    it "Returns true if the product is removed" do
+      products = { "banana" => 1.99, "cracker" => 3.00 }
+      order = Grocery::Order.new(1337, products)
+
+      result = order.remove_product("banana")
+      result.must_equal false
+    end
+
+    it "Returns true if the product is removed" do
+      products = { "banana" => 1.99, "cracker" => 3.00 }
+      order = Grocery::Order.new(1337, products)
```

Comment:
You forgot to change the name of this test when you copy/pasted.

Reply:
Thanks for catching it. Changed.